sktime/utils/validation/forecasting.py

```diff
@@ -328,13 +328,24 @@ def check_scoring(scoring):
     ----------
     TypeError
         if object is not callable from current scope.
+    NotImplementedError
+        if metric requires y_pred_benchmark to be passed
     """
     # Note symmetric=True is default arg for MeanAbsolutePercentageError
     from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
         return MeanAbsolutePercentageError()
 
+    if scoring._tags["requires_y_pred_benchmark"]:
```

Comment:
I think we have a `_has_tag(...)` or `has_tag(...)` function in utils somewhere for this, makes it a bit more readable.